Sources/Starknet/Data/Transaction/TransactionHash.swift

```diff
@@ -1,16 +1,59 @@
 import Foundation
 
 class TransactionHashCalculator {
```

Comment:
Can we make this entity public or provide the methods to compute the hash in `StarknetAccountProtocol`? It could be useful for us to know the computed hash before submitting the transaction in certain scenarios.

Reply:
Sure, done